.github/workflows/bump_version.yaml

```diff
@@ -0,0 +1,25 @@
+name: 'Test upload Dockerfile'
+
+on:
+  push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ github.token }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: create fake version file
+        run: |
+          echo "VERSION=1.2.3" > VERSION
+      - uses: ./xcore/bump_version_pull_request
+        id: pull-request
```

Comment:
may be we want to check that VERSION is 1.2.4 now ?

Reply:
yes done